tests/integration/services/activation/test_activation.py

```diff
@@ -0,0 +1,125 @@
+#  Copyright 2023 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Activation Manager tests."""
+# TODO(alex) dedup code and fixtures across all the tests
+
+import pytest
+
+from aap_eda.core import models
+
+PROJECT_GIT_HASH = "684f62df18ce5f8d5c428e53203b9b975426eed0"
+
+
+@pytest.fixture
+def default_rulebook() -> models.Rulebook:
```

Comment:
What if all these fixtures are located in a conftest.py for reusability?

Reply:
Created conftest.py for common fixtures.